src/Sdk.php

```diff
@@ -9,13 +9,23 @@
 /**
  * Creates a new Client and Hub which will be set as current.
  *
- * @param array $options The client options
+ * @param array $arrayOptions The client options
  */
-function init(array $options = []): void
+function init(array $arrayOptions = []): void
 {
-    $client = ClientBuilder::create($options)->getClient();
+    $options = new Options($arrayOptions);
+    $client = (new ClientBuilder($options))->getClient();
```

Comment:
Why you changed this to `new` instead of using the static method `create` which makes the code nicer?